src/Client.php

```diff
@@ -154,7 +154,7 @@ function (CommandInterface $req, CommandInterface $res) use ($handle) {
         );
     }
 
-    protected function handleWorkEvent(CommandInterface $command)
+    public function handleWorkEvent(CommandInterface $command)
```

Comment:
I'll assume this is related to your testing of the `WORK_FAIL`-issue (#7). Can your revert the signature change and squash (`git rebase -i HEAD~3`, https://www.devroom.io/2011/07/05/git-squash-your-latests-commits-into-one/) your commits into a single one? Thanks! :smile: